src/app/data/services/data_service.spec.ts

```diff
@@ -39,6 +39,8 @@ describe('DataService', function () {
       service.loadContent(contentBlock, false).then((block:ContentBlock) => null);
       expect(ajaxService.fetchContentBlocks).toHaveBeenCalledWith(contentBlock.apiKey, contentBlock.apiHost, false);
       expect(spy.calls.count()).toBe(1);
+      service.loadContent(contentBlock, false);
+      expect(spy.calls.count()).toBe(2);
```

Comment:
this should be 1, not because of the http cache, but because it's the same ajax request. added here: https://github.com/renuo/renuo-cms-client/blob/feature/5029-55-73-ajax-request/src/app/data/services/data_service.spec.ts#L64